mixer/adapter/stackdriver/config/config.proto

```diff
@@ -73,7 +73,9 @@ message Params {
     // Describes how to represent an Istio metric in Stackdriver.
     // See https://github.com/googleapis/googleapis/blob/master/google/api/metric.proto
     message MetricInfo {
+        // The kind of measurement for metric. It describes how the data is reported. Ex: Gauge.
```

Comment:
for metric. It describes -> for a metric, which describes

Reply:
Fixed